crates/services/upgradable-executor/src/executor.rs

```diff
@@ -1111,7 +1107,6 @@ mod test {
         // If it doesn't cache the modules, the test will fail with a timeout.
         #[test]
         #[cfg(not(coverage))] // Too slow for coverage
-        #[ntest::timeout(60_000)]
```

Comment:
Ok I see, now then I will try to find another way to check if we hit the cache otherwise I will readd it

Reply:
Readded with a bit bigger timeout given that on the last CI runs it took 50 seconds

Reply:
Even with 90 seconds it can trigger the timeout : https://github.com/FuelLabs/fuel-core/actions/runs/11812498813/job/32907837216?pr=2401#step:8:2825

I have changed it to only compute time for the calls that expect to use cache